examples/SingleParticleCurrent/include/particles/ParticlesInitOneParticle.hpp

```diff
@@ -73,7 +73,10 @@ __global__ void kernelAddOneParticle(ParBox pb,
                 SetAttributeToDefault<bmpl::_1> > setToDefault;
             setToDefault(forward(par));
         }
-        float3_X pos = float3_X(LOCAL_POS_X, LOCAL_POS_Y, LOCAL_POS_Z);
+        float3_X posTmp = float3_X(LOCAL_POS_X, LOCAL_POS_Y, LOCAL_POS_Z);
```

Comment:
please write `floatD_X pos = float3_X(LOCAL_POS_X, LOCAL_POS_Y, LOCAL_POS_Z).shrink<simDim>();`
might be `floatD_X pos = float3_X(LOCAL_POS_X, LOCAL_POS_Y, LOCAL_POS_Z).template shrink<simDim>();`